Library/OptimizelyFeature.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 
 public enum OptimizelyFeature {
   public enum Key: String {
+    case commentThreading = "ios_comment_threading"
     case lightsOn = "ios_lights_on"
```

Comment:
Pretty sure we can remove lights on, but should do that in a separate PR if we do.

Reply:
I'll raise another PR with this, in a moment.